app.py

```diff
@@ -13,35 +13,35 @@
     MessageEvent, TextMessage, TextSendMessage,
 )
 
+import nlu
+
 app = Flask(__name__)
 
 line_bot_api = LineBotApi(os.environ.get('LINE_CHANNEL_ACCESS_TOKEN'))
 handler = WebhookHandler(os.environ.get('LINE_CHANNEL_SECRET'))
 
+
 @app.route("/callback", methods=['POST'])
 def callback():
     # get X-Line-Signature header value
     signature = request.headers['X-Line-Signature']
-
     # get request body as text
     body = request.get_data(as_text=True)
-    app.logger.info("Request body: " + body)
-
-    # handle webhook body
     try:
         handler.handle(body, signature)
     except InvalidSignatureError:
         print("Invalid signature. Please check your channel access token/channel secret.")
         abort(400)
-
     return 'OK'
 
+
 @handler.add(MessageEvent, message=TextMessage)
 def handle_message(event):
-    text = jieba.cut(event.message.text)
+    text = nlu.execute(event.message.text)
```

Comment:
execute absolutely not a good name here, rename to something like response_message

Reply:
you're right again.